etcdctl/command/role_commands.go

```diff
@@ -101,7 +101,7 @@ func actionRoleList(c *cli.Context) {
 	}
 
 	for _, role := range roles {
-		fmt.Printf("%s\n", role)
+		fmt.Printf("%s\n", role.Role)
```

Comment:
I would prefer to fix them in follow-up PRs to make this one focus on one thing.